exercises/strain/HINTS.md

```diff
@@ -0,0 +1 @@
+`apply` will let you pass arguments to a function
```

Comment:
Or, maybe we could also include the other way that you can call an anonymous function: `fun.(head)`, which is the same in this case as `apply`, and probably a little more beginner friendly for this early in the track.